src/Moryx.Products.Management/Model/Migrations/Sqlite/20251202123359_StepTypeName.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
We should still add the license header in the auto-generated files, don't you think? 🤔